### PR TITLE
!status

### PR DESCRIPTION
**Pull request by LW001**
I've finally added !status. yay.
What differenciates this from my other stuff is that it's [fully tested](https://dabbit.uservoice.com/forums/598597-general/suggestions/19700419-hey#{toggle_previous_statuses}).

How to use:
**!status** _URL/Link/UVRegex_ _Name of status flag or none to not set a status_ _Message to add to it_

Status flag should work with the name of any flag, if it's not a valid flag name there just won't be a status changed but still a response created.

### CODE REVIEW

File: Commands/Commands/uservoice.js [hunk 1]
```diff
@@ -240,6 +240,107 @@ commands.comment = {
...
+        }).then(data => {
+          status = status.toLowerCase().split(' ').map(function(word) {
+            return word.replace(word[0], word[0].toUpperCase());
+          }).join(' ');
+          msg.reply('status was updated.', false, {
+            title: entities.decode(data.suggestion.title),
+            url: data.suggestion.url,
+            description: (data.suggestion.text !== null) ? ((data.suggestion.text.length < 1900) ? entities.decode(data.suggestion.text) : '*Content too long*') : '*No content*',
+            color: 0x3498db,
+            author: {
```
Comment (Dougley) [inline Commands/Commands/uservoice.js:278]: A creator object is not guaranteed (loluv)

**Review by Dougley — CHANGES_REQUESTED** [review]

**Comment by Dougley** [comment]
Also, what would the exact syntax be? Please provide a working example.

**Comment by LW001** [comment]
`!status https://dabbit.uservoice.com/forums/598597-general/suggestions/20603587-create-a-status-reaction planned This will be added at some point.`

Probably will be changed to
`!status https://dabbit.uservoice.com/forums/598597-general/suggestions/20603587-create-a-status-reaction planned | This will be added at some point` though

**Comment by LW001** [comment]
New usage
`!status 19700419 | completed | we needed pipes`

**Comment by RSNFreud** [comment]
*wonders how long this will take to get passed*

**Comment by RSNFreud** [comment]
Also mandatory **LWWarez** flag.

**Comment by LW001** [comment]
Yo what's the plan with this now?

**Comment by Dougley** [comment]
soon™️, haven't had the time to properly review.

**Comment by RSNFreud** [comment]
1 month later...

**Comment by Arzte** [comment]
@RSNFreud please stop adding comments like this, we are aware of all the pull requests, we are currently working on a full rewrite of the bot and therefore are holding of on most pr's for the moment.

**Comment by RSNFreud** [comment]
Ahh sorry. Wasnt aware of that cause.. yknow.. banned

**Pull request closed without merging**